packages/ifpack2/src/Ifpack2_Details_Chebyshev_def.hpp

```diff
@@ -1417,6 +1425,44 @@ powerMethodWithInitGuess (const op_type& A,
     *out_ << "  norm1(x.scale(one/norm)): " << x.norm1 () << endl;
   }
 
+  /*
+    We use a heuristic for stopping early if tolerance is specified.
+    It works the following way:
+
+    Lets assume that
+      (a) our estimate for lambdaMax increases monotonically.
```

Comment:
Question: this would still not show up in the Doxygen documentation, right? Should I move it to the `Ifpack2_Chebyshev_decl.hpp` file for the description of the tolerance parameter?